api/Document.json

```diff
@@ -9756,40 +9756,40 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-readystatechange",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "9"
```

Comment:
I would bet real money that this is also a case of the event being supported earlier than the `onreadystatechange` property which this is mirroring data from. It doesn't matter much so if we had a range we could just drop that in, but I expect this is wrong.

Fortunately this is very easy to test, can you try http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9673 in older browsers?